### PR TITLE
Policy updates & HALL OF FAME

### PR DESCRIPTION
**Pull request by KristianGrafana**
* Adding renniepak in the Hall of Fame
* Add CSRF to out-of-scope

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Valid vulnerabilities on any product not explicitly listed in scope will be acce
     * Secure coding best practice not used
     * TLS configuration with older ciphersuites
     * Host enumeration (e.g. via Semi-blind SSRF)
-    * Logout CSRF
+    * CSRF
```
Comment (jmatosgrafana) [inline README.md:59]: What about excluding "CSRF with only an Availability impact" ?

File: HALL_OF_FAME.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 # Hall of fame
 
 Grafana Labs would like to thank the following security researchers for having reported those vulnerabilities:
+
+[renniepak](https://github.com/renniepak) - 💰💰
```
Comment (jmatosgrafana) [inline HALL_OF_FAME.md:5]: Do we want to list the corresponding CVEs ?

File: README.md (hunk 1 above)
Comment (KristianGrafana) [reply README.md]: That's reasonable, see https://github.com/grafana/bugbounty/pull/9/commits/ee906f2b68d26f4673613ccc70ff64e8114efc27 for that change.

File: HALL_OF_FAME.md (hunk 2 above)
Comment (KristianGrafana) [reply HALL_OF_FAME.md]: That's a good idea, here's a example of that could look like: https://github.com/grafana/bugbounty/pull/9/commits/1a12991ebbe8ec646291777db9a6675b3f2e9bee

**Pull request merged**